src/node_options.h

```diff
@@ -159,7 +159,7 @@ class EnvironmentOptions : public Options {
   bool trace_warnings = false;
   bool extra_info_on_fatal_exception = true;
   std::string unhandled_rejections;
-  std::string userland_loader;
+  std::vector<std::string> userland_loaders;
```

Comment:
FYI reverting this line (and the related change in `src/node_options.cc`) seems to fix the "async_hooks thing".